### PR TITLE
move velvet from devteam

### PR DESCRIPTION
**Pull request by bernt-matthias**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Comment by bgruening** [comment]
We need someone that grants IUC access to the repos. ping @martenson

https://github.com/galaxyproject/tools-iuc/runs/766533966?check_suite_focus=true

**Comment by martenson** [comment]
granted, restarted, thanks for the ping 👍

File: tools/velvet/.shed.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+- Assembly
+description: 'de novo genomic assembler specially designed for short read sequencing technologies'
+homepage_url: https://www.ebi.ac.uk/~zerbino/velvet/
+long_description: |
+  Velvet currently takes in short read sequences, removes errors then produces
+  high quality unique contigs. It then uses paired-end read and long read
+  information, when available, to retrieve the repeated areas between contigs.
+name: velvet
+owner: devteam
+remote_repository_url: https://github.com/galaxyproject/tools-devteam/tree/master/tools/velvet
```
Comment (nsoranzo) [inline tools/velvet/.shed.yml:11]: This should have been updated to https://github.com/galaxyproject/tools-iuc/tree/master/tools/velvet

Comment (bernt-matthias) [reply tools/velvet/.shed.yml:11]: https://github.com/galaxyproject/tools-iuc/pull/3083

**Pull request merged**